catalogue/webapp/pages/works.js

```diff
@@ -207,6 +239,7 @@ const Works = ({ works, images, worksRouteProps, apiProps }: Props) => {
                     <div className="flex flex--h-space-between flex--v-center">
                       <Fragment>
                         <Paginator
+                          query={query}
```

Comment:
We're happy for this to leak into prod?

Reply:
I was going to ask about that. Probably safest to wrap it in the toggle for now.